website/docs/r/available_ip_address.md

```diff
@@ -67,16 +76,15 @@ resource "netbox_available_ip_address" "myvm-ip" {
 
 ### Required
 
-- **prefix_id** (String)
-- **status** (String)
```

Comment:
The status is still `required` as per the schema above

Reply:
Meant to make status optional.  Have done so now, with default being "active".  Is non-breaking.